lua/nvim-tree/actions/open-file.lua

```diff
@@ -194,6 +194,15 @@ function M.fn(mode, filename)
   local do_split = mode == "split" or mode == "vsplit"
   local vertical = mode ~= "split"
 
+  -- Check if file is already loaded in a buffer
+  local buf_loaded = false
+  for _, bufId in ipairs(api.nvim_list_bufs()) do
```

Comment:
bufId should be buf_id

Reply:
Thanks, changed. 
Javascript influence;)